### PR TITLE
IOR Resolver

### PR DESCRIPTION
**Pull request by mathersm-dst**

### CODE REVIEW

File: TAF/taf/IORResolver_T.cpp [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+***************************************************************/
+#ifndef TAF_IORRESOLVER_T_CPP
+#define TAF_IORRESOLVER_T_CPP
+
+
+#include "IORResolver_T.h"
+#include <taf/ORBManager.h>
+
+
+#define IOR_FILENAME_EXTENSION ACE_TEXT(".ior");
```
Comment (jwillemsen) [inline TAF/taf/IORResolver_T.cpp]: ACE_TEXT is only for ACE_TCHAR, this is used with a std::string so ACE_TEXT shouldn't be used. Maybe make this a const std::string?

File: TAF/taf/IORResolver_T.cpp [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+    FileResolver_T<T>::~FileResolver_T(void)
+    {
+    }
+
+
+    template <typename T>
+    int
+    FileResolver_T<T>::run(void)
+    {
+        CORBA::Object_var obj(0);
```
Comment (jwillemsen) [inline TAF/taf/IORResolver_T.cpp]: _var is empty initialized, shouldn't use 0 for a CORBA pointer

File: TAF/taf/IORResolver_T.cpp [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+                        if (TAF::debug())
+                        {
+                            ACE_DEBUG((LM_INFO, ACE_TEXT("TAF (%P|%t) FileResolver_T::run - Resolved %C from %C\n"), this->name_.c_str(), filename.c_str()));
+                        }
+                        return 0;
+                    }
+                    catch (const CORBA::Exception &ce)
+                    {
+                        ACE_UNUSED_ARG(ce);
+                        ACE_ERROR((LM_ERROR, ACE_TEXT("TAF (%P|%t) FileResolver_T::run - CORBA Exception trying to resolve %C from %C\n"), this->name_.c_str(), filename.c_str()));
```
Comment (jwillemsen) [inline TAF/taf/IORResolver_T.cpp]: Would help users when you also log the _info of the exception

File: TAF/training/CORBActiveService/CORBActiveClient.cpp [hunk 4]
```diff
@@ -106,6 +109,16 @@ namespace { // Anonomous Namespace
     template <typename T>
     typename T::_var_type  resolveCORBAService(const char *name, bool use_naming = true)
     {
+#if 1
```
Comment (jwillemsen) [inline TAF/training/CORBActiveService/CORBActiveClient.cpp]: Do you need to keep the old code, you have git for history

**Comment by mathersm-dst** [comment]
Addressed issue #86

**Pull request merged**